showcase/src/app/app.component.html

```diff
@@ -120,16 +122,15 @@
 					</div>
 				</div>
 			</header>
-			<div class="stark-main-container">
-				<main>
-					<ui-view></ui-view>
-				</main>
-			</div>
+			
+			<main class="stark-main-container">
```

Comment:
Could you please document this needed change in your commit message (as it is a breaking change) ? 😊

Reply:
```Markdown
  - the css selector for `.stark-main-container` has changed and can now be applied directly to the `main` element in app.component.html.
  (see Starter for an example)
```